docs/index.html

```diff
@@ -7,56 +7,49 @@
 <div class="panel panel-info">
     <div class="panel-heading">
         {% include icon.html type="info" %}
-        ev3dev is a community-maintained project that relies on people's contributions.
+        ev3dev is a community-maintained project that relies on community contributions.
     </div>
     <div class="panel-body">
         <p>
-            If there's a
-            topic that you think we should have docs for, please consider
-            <a href="../contribute">contributing</a>.
+            If there's a topic that you think we should have docs for, please
+            consider <a href="../contribute">contributing</a>.
         </p>
         <p>
             <a href="https://github.com/ev3dev/ev3dev/issues/153#issuecomment-93602107">Some pages</a>
-            have not been migrated from our old wiki yet. We'd love some help with getting these last few pages migrated:
-            if you would like to help out, read more about what's needed in the <a href="../contribute#write-some-documentation">documentation section</a>
+            have not been migrated from our old wiki yet. We'd love some help
```

Comment:
> Actually, ev3dev stickers might be good reward.

Oh, good idea! I recall Ralph offered them to me on multiple occasions but there was an unfortunate disconnect between me putting in on my to-do list and it actually getting done :laughing: Something as simple as stickers like that would be a nice incentive while still only costing a minimal amount.